{{cookiecutter.project_slug}}/README.md

```diff
@@ -34,29 +39,33 @@ this will build the entire project with all dependencies inside a docker contain
 
 We use `pytest` as test framework. To execute the tests, please run
 
-    python setup.py test
+    {% if cookiecutter.package_manager == 'poetry' %}poetry run pytest tests{% else %}python setup.py test{% endif %}
 
 To run the tests with coverage information, please use
 
-    python setup.py testcov
+    {% if cookiecutter.package_manager == 'poetry' %}poetry run pytest tests --doctest-modules --junitxml=junit/test-results.xml --cov=src --cov-report=xml --cov-report=html{% else %}python setup.py testcov{% endif %}
 
 and have a look at the `htmlcov` folder, after the tests are done.
 {% if cookiecutter.use_notebooks == 'yes' %}
 ### Notebooks
-
+{% if cookiecutter.package_manager == 'poetry' %}
+You can use your module code (`src/`) in Jupyter notebooks (`notebooks/`) without running into import errors by running:
+    poetry run jupyter notebook
```

Comment:
I thin you need a newline before this one or it won't render as code.

Also, can you explain how this works? This this open a jupyter lab instance or something?
Could you add a short explanation below the command?

Reply:
Sure, can do